src/index.ts

```diff
@@ -1 +1,2 @@
 export { getManagementToken } from './keys'
+export { createSignature } from './requests'
```

Comment:
Not sure if we should reveal this method or rather two methods `generateSignature, verifcySignature`. Nbd for now at least

Reply:
What would the difference between `generateSignature` and `createSignature`? For the other method on Friday you suggested to do it separately 😅 Shall we do it here?

Reply:
I'm terrible with naming here but I was imagining two exposed methods:
- generating the signature(returns headers object): actually creates the headers (e.g: timestamp, signing-order, signature) : essentially the method you run the first time
- verifying the signature (returns a boolean/throws on mismatch): like it says on the tin

Both methods would use `createSignature` that would always just return the signature as named here but the business logic behind each

Reply:
Maybe it's better to handle generating the headers in the call site where we use this? 

I think verifying the signature is going to be implemented in a separate PR

Reply:
I agree with @Jwhiles here. Added also the verification to next commits